<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WazeObserverClass</class>
 <widget class="QMainWindow" name="WazeObserverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WazeObserver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="latLineEdit"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="descrLineEdit"/>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lonLineEdit"/>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QListWidget" name="reportList"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="validateButton">
      <property name="text">
       <string>VALIDATE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="WazeObserver.qrc"/>
 </resources>
 <connections/>
</ui>
